src/runtime/vm/vm.cc

```diff
@@ -569,20 +570,36 @@ std::ostream& operator<<(std::ostream& os, const VMFunction& vm_func) {
   return os;
 }
 
+Object CopyTo(Object src, const DLContext& ctx, bool force = false) {
```

Comment:
Add a comment about why we need `force`, i.e. when it is used? It looks we don't use it in this PR.